uelc/main/models.py

```diff
@@ -581,6 +581,13 @@ def display_title(self):
     def display_description(self):
         return self.description
 
+    @classmethod
+    def add_form(self):
```

Comment:
The first argument of a `classmethod` is a reference to the class, not an instance. It's conventional to call it `cls` instead of `self` to avoid confusion: https://docs.python.org/2/library/functions.html#classmethod